aiomcache/client.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, host, port=11211, *,
     # key supports ascii sans space and control chars
     # \x21 is !, right after space, and \x7e is -, right before DEL
     # also 1 <= len <= 250 as per the spec
-    _valid_key_re = re.compile(b'^[\x21-\x7e]{1,250}$')
+    _valid_key_re = re.compile(b'^[^\x00-\x20\x7f]{1,250}$')
```

Comment:
As per the above comment and the memcached documention, "the key must not include
control characters or whitespace."

Therefore this will need some more work to allow non-ascii characters.